src/network/invthread.py

```diff
@@ -42,7 +42,7 @@ def handleLocallyGenerated(stream, hashId):
         """Locally generated inventory items require special handling"""
         dandelion.instance.addHash(hashId, stream=stream)
         for connection in connectionpool.pool.connections():
-            if state.dandelion_enabled and connection != \
+            if dandelion.instance.enabled and connection != \
```

Comment:
here, `objectChildStem` could return value depending on whether dandelion is enabled or not